toolkit/cli/smart-contracts-commands/src/governance.rs

```diff
@@ -58,9 +62,15 @@ impl InitGovernanceCmd {
 pub struct UpdateGovernanceCmd {
 	#[clap(flatten)]
 	common_arguments: crate::CommonArguments,
+	/// Old governance authority hash to be set.
+	#[arg(short = 'o', long, num_args = 1.., value_delimiter = ' ')]
+	old_governance_authority: Vec<MainchainKeyHash>,
 	/// Governance authority hash to be set.
-	#[arg(long, short = 'g')]
-	new_governance_authority: MainchainKeyHash,
+	#[arg(short = 'g', long, num_args = 1.., value_delimiter = ' ')]
+	new_governance_authority: Vec<MainchainKeyHash>,
+	/// Governance threshold to be set.
```

Comment:
What is goveranance threshold? This UI change should be reflected in docs changes.